docker/packager/packager

```diff
@@ -90,7 +90,7 @@ def run_docker_image_with_env(
         user = f"{os.geteuid()}:{os.getegid()}"
 
     cmd = (
-        f"docker run --network=host --user={user} --rm --volume={output}:/output "
+        f"docker run -v /var/run/docker.sock:/var/run/docker.sock --network=host --user={user} --rm --volume={output}:/output "
```

Comment:
We need to add a comment explaining that this is required for docker-in-docker required by the way we build BoringSSL libraries in FIPS mode.